modules/rest-api/pages/rest-buckets-streamingURI.adoc

```diff
@@ -1,5 +1,6 @@
 = Getting Bucket Streaming URI
 :page-topic-type: reference
+:page-aliases: rest-buckets-streaminguri
```

Comment:
This one seems to be correcting for the fact that the old site didn't support case-sensitive URLS?

Reply:
Right, it's just correcting for the lower-case uri. But it needs fixing with the current site, apparently.